lua/lvim/core/nvimtree.lua

```diff
@@ -123,16 +137,22 @@ function M.setup()
     vim.g.netrw_banner = false
   end
 
+  local function telescope_find_files(_)
+    require("lvim.core.nvimtree").start_telescope "find_files"
+  end
+  local function telescope_find_str(_)
+    require("lvim.core.nvimtree").start_telescope "live_grep"
+  end
+
   -- Add useful keymaps
-  local tree_cb = nvim_tree_config.nvim_tree_callback
   if #lvim.builtin.nvimtree.setup.view.mappings.list == 0 then
     lvim.builtin.nvimtree.setup.view.mappings.list = {
-      { key = { "l", "<CR>", "o" }, cb = tree_cb "edit" },
-      { key = "h", cb = tree_cb "close_node" },
-      { key = "v", cb = tree_cb "vsplit" },
-      { key = "C", cb = tree_cb "cd" },
-      { key = "gtf", cb = "<cmd>lua require'lvim.core.nvimtree'.start_telescope('find_files')<cr>" },
-      { key = "gtg", cb = "<cmd>lua require'lvim.core.nvimtree'.start_telescope('live_grep')<cr>" },
+      { key = { "l", "<CR>", "o" }, action = "edit", mode = "n" },
+      { key = "h", action = "close_node" },
+      { key = "v", action = "vsplit" },
+      { key = "C", action = "cd" },
+      { key = "gtf", action_cb = telescope_find_files },
+      { key = "gtg", action_cb = telescope_find_str },
```

Comment:
```suggestion
      { key = "gtf", action = "telescope_find_files", action_cb = telescope_find_files },
      { key = "gtg", action = "telescope_live_grep", action_cb = telescope_live_grep },
```
Apparently the `action` tag is needed for some reason..